README.md

```diff
@@ -36,6 +36,10 @@ Via pecl/pickle:
 install-php-extensions opentelemetry[-beta|-stable|-latest]
 ```
 
+### Windows
+
+See https://wiki.php.net/internals/windows/stepbystepbuild_sdk_2#building_pecl_extensions_with_phpize
```

Comment:
Have not tried that for some time, however I'm not sure if this instruction is valid

Reply:
That's what the github action does, and it seems to work (compiles, and the tests all run and pass). I copied most of the steps from phpredis extension's windows build.